tools/toolchain/scripts/generate_arch_files.sh

```diff
@@ -396,14 +396,14 @@ EOF
 FYPPFLAGS   = -n --line-marker-format=gfortran5
 EOF
   fi
-  if [ "${with_intel}" != "__DONTUSE__" ]; then
+  if [ "${with_ifx}" != "no" ]; then
```

Comment:
This is a mistake and wrong. I wanted to apply workaround only for IFORT. There will be some follow-on PR...